datasets/swiss_judgment_prediction/README.md

```diff
@@ -153,11 +156,11 @@ The following data fields are provided for documents (`train`, `dev`, `test`):
 
 ### Data Splits
 
-| Language |  ISO code | Number of Documents (Training/Dev/Test) | 
-| ---- | ---- |  ---- |  
-| German    | **de**   | 35'452 / 4'705 / 9'725
-| French    | **fr**   | 21'179 / 3'095 / 6'820
-Italian     | **it**   | 3'072 / 408 / 812
+| Language | ISO code | Number of Documents (Training/Validation/Test) | 
+|---------|----------|------------------------------------------------|  
+| German  | **de**   | 35'452 / 4'705 / 9'725                         |
+| French  | **fr**   | 21'179 / 3'095 / 6'820                         |
+| Italian | **it**   | 3'072 / 408 / 812                              |
```

Comment:
This table should be updated with the new configurations (languages).

Reply:
I updated it